CHANGELOG.md

```diff
@@ -22,6 +23,7 @@ All notable changes to this project will be documented in this file.
 -   Allow printing the result of a function returning multiple objects of different types.
 -   #1792 : Fix array unpacking.
 -   #1795 : Fix bug when returning slices in C.
+-   #1732 : Fix multidimensional list indexing.
```

Comment:
```suggestion
-   #1732 : Fix multidimensional list indexing in Python.
```
This is not entirely true, but will be soon

Reply:
updated 074745b3043674dbb60b6b6bf5eaa03fa6036df7